### PR TITLE
Add logging to add command code

### PR DESCRIPTION
**Pull request by kailashgautham**

### CODE REVIEW

**Review by YuSoonZ — APPROVED** [review]
LGTM!

**Review by teojunda — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -41,6 +44,7 @@ public class AddCommand extends Command {
     public static final String MESSAGE_DUPLICATE_FIELD = "This person contains duplicate fields"
             + "(Email, Phone, Telegram, or Github) with another person.";
 
+    private static final Logger logger = Logger.getLogger("Foo");
```
Comment (g-tejas) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:47]: Shouldn't this be Logger.getLogger(AddCommand.class)

**Review by g-tejas — CHANGES_REQUESTED** [review]
LGTM except for 1 small clarification

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 1 above)
Comment (kailashgautham) [reply src/main/java/seedu/address/logic/commands/AddCommand.java]: Fixed, thanks!

**Review by g-tejas — APPROVED** [review]
LGTM

**Pull request merged**